src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--5 div {
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead of selecting `div` directly within `.stars--5`, use a class that you assign to the children of `.stars--5`.